lib/baw-workers/redis_communicator.rb

```diff
@@ -0,0 +1,122 @@
+module BawWorkers
+  class DecodeException < StandardError;
```

Comment:
What's the reasoning for moving to the error module?

I left it here because it came with the code I blatantly lifted from Resque.